justspaces/settings.py

```diff
@@ -138,7 +138,7 @@
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 STATIC_URL = '/static/'
 
-LOGIN_REDIRECT_URL = 'home'
+LOGIN_REDIRECT_URL = 'survey-list'
```

Comment:
I think this makes the most sense! (Later, we'll need to decide if this redirect should be to "editable" or "run-able" surveys.)